XamarinCommunityToolkit/Helpers/LocalizationResourceManager.shared.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.ComponentModel;
+using System.Globalization;
+using System.Resources;
+using System.Threading;
+
+namespace Xamarin.CommunityToolkit.Helpers
+{
+#if !NETSTANDARD1_0
+    public class LocalizationResourceManager : INotifyPropertyChanged
+    {
+        public static LocalizationResourceManager Current { get; } = new LocalizationResourceManager();
+
+        ResourceManager resourceManager;
+
+        public void Init(ResourceManager resource)
+		{
```

Comment:
missing one tab here

Reply:
I think is a Github preview file issue. If you go to the file directly it looks good.
https://github.com/xamarin/XamarinCommunityToolkit/blob/26a7a58901a89baad3963a8070b5395a731569ab/XamarinCommunityToolkit/Helpers/LocalizationResourceManager.shared.cs